style/main.css

```diff
@@ -44,9 +44,9 @@ header nav a:hover{
 
 .filme-principal{
     font-size: 16px;
-    background: linear-gradient(rgba(0,0,0,.50),rgba(0,0,0,.50)100%), url('/img/ate_o_ultimo_h\ -\ capa.jpg');
+    background: linear-gradient(rgba(0,0,0,.50),rgba(0,0,0,.50)100%), url('../img/ate_o_ultimo_h_capa.jpg');
 
-    height: 400px;
+    min-height: 400px;
```

Comment:
Colocar uma altura fixa pode causar problemas. O que acontece se o texto do `.filme-principal` for tão longo que gera tantas linhas que o parágrafo fica com mais do que 400px de altura? Para evitar esse tipo de problema, usa-se `min-height` que garante que o elemento nunca vai ser mais baixo que o valor definido mas deixa o elemento crescer para acomodar mais conteúdo.